integration_tests/backend/setup/flow_with_failure.py

```diff
@@ -11,12 +11,12 @@ def setup_flow_with_failure(client: aqueduct.Client, integration_name: str) -> s
     n_runs = 1
     integration = client.integration(name=integration_name)
 
-    @aqueduct.op()
+    @aqueduct.op(requirements=[])
     def bad_op(df):
         x = y  # intentional buggy code
         df["new"] = df["review"]
 
-    @aqueduct.op()
+    @aqueduct.op(requirements=[])
```

Comment:
Any reason we can't just add `requirements.txt` to `integration_tests/backend/`? This seems like a hassle.